resources/page/permalinks_test.go

```diff
@@ -69,6 +69,7 @@ func TestPermalinkExpansion(t *testing.T) {
 	page.date = d
```

Comment:
This all looks good, but I think we need a test that shows the new setup working. I suggest you create a new `permalink_integration_test.go` and follow the the cue of similar tests (using the NewIntegrationTest...) that tests the new permalink setup in a site build.